README.md

```diff
@@ -3,6 +3,11 @@
 
 ![Egeria Logo](assets/img/ODPi_Egeria_Logo_color.png)
 
+[![GitHub](https://img.shields.io/github/license/odpi/egeria)](LICENSE.md)
+[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/3044/badge)](https://bestpractices.coreinfrastructure.org/projects/3044)
+[![Azure](https://dev.azure.com/odpi/egeria/_apis/build/status/odpi.egeria)](https://dev.azure.com/odpi/Egeria/_build)
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=odpi_egeria&metric=alert_status)](https://sonarcloud.io/dashboard?id=odpi_egeria)
+
```

Comment:
I see chris commented on this already. Obviously we define the gate, I think we all know we have more work to do here. I would be ok about including it though - we should be transparent, and welcome help!